src/Components/Wizard/Wizard.tsx

```diff
@@ -155,6 +155,7 @@ export class Wizard extends React.Component<WizardProps, WizardState> {
         initialValues={initialValues}
         validate={validate}
         validationSchema={validationSchema}
+        validateOnChange={false}
         onSubmit={this.handleSubmit}
```

Comment:
Is this built into the form library? I don't see it used.

Reply:
Yeah, it comes from formik https://github.com/jaredpalmer/formik#validateonchange-boolean